guides/doc-Provisioning_Guide/topics/Virt-VMWare.adoc

```diff
@@ -406,6 +406,16 @@ You can now create an image from this virtual machine.
 
 You can use the xref:Provisioning_Virtual_Machines_in_VMware_vSphere-Adding_VMware_vSphere_Images_on_the_Satellite_Server[] section to add the image to {Project}.
 
+.Configuring {SmartProxy} to forward user data template
+
+{Project} must recognize the client IP address in order to serve correct template payload and when clients connects directly to {ProjectServer}, remote IP of the HTTP request can be used. However, when {Project} is deployed with {SmartProxy} templates feature, the callback URL is set to {SmartProxy} endpoint which forwards to the {ProjectServer} with HTTP header information X-Forwarded-For.
+
+For security reasons, {Project} does recognize this HTTP header only from localhost. For each individual {SmartProxy}, a regular expression in *Administer* > *Settings* > *Provisioning* > *Remote address* must be updated to recognize IP address. From CLI this can be done using the following command:
```

Comment:
```suggestion
For security reasons, {Project} only accepts this HTTP header from localhost. For each individual {SmartProxy}, a regular expression in *Administer* > *Settings* > *Provisioning* > *Remote address* must be updated to recognize IP address. From CLI this can be done using the following command:
```